requirements/deploy.txt

```diff
@@ -398,11 +398,11 @@ sqlparse==0.4.3
     #   django-debug-toolbar
 stack-data==0.6.1
     # via ipython
-stripe==4.2.0
+stripe==5.0.0
```

Comment:
There are some breaking changes that don't affect us: https://github.com/stripe/stripe-python/blob/master/CHANGELOG.md#500---2022-11-16

We could go one step further and start using `.modify` instead of `.save` where used, to be prepared for it when it gets removed. Now, it's deprecated.

Reply:
@stsewd would be good if you can double-check this in case I missed something from the changelog

Reply:
We aren't using `.save` on resources. The only thing that could be a problem is with djstripe, they don't have an upper bound for stripe https://github.com/dj-stripe/dj-stripe/blob/88c42abf403170493185de3cdb260f222e6951db/pyproject.toml#L46

Actually, it can break djstripe, yeah https://github.com/dj-stripe/dj-stripe/issues/1842. We can't upgrade till djstripe removes the deprecated stripe.Order stuff.

Reply:
Cool! I'm pinning `stripe` and adding a comment for this 👍🏼